newIDE/app/src/UI/SearchBar.js

```diff
@@ -163,7 +161,6 @@ export default class SearchBar extends React.PureComponent<Props, State> {
           ...styles.root,
           ...style,
         }}
-        elevation={this.props.elevation || 1}
```

Comment:
I'm confused why we're seeing this in the PR? oO

Reply:
Okay I didn't see this. All of the changes from different PRs are shown in my commit related to merging the master. 🙄 This is odd.

Reply:
Ah right I've seen this happen already, might be a GitHub bug. Closing this PR and re-opening another one should fix this :)